about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] Your about page looks distorted. Therefore, I could not check against the required changes raised by the last reviewer. Kindly ensure that all links are fixed before requesting another review. 

![about](https://user-images.githubusercontent.com/26856491/124123946-c23dff00-da6f-11eb-9827-532e1f63cc78.gif)